Sources/Sentry/SentryScope.m

```diff
@@ -50,6 +51,7 @@ - (instancetype)initWithMaxBreadcrumbs:(NSInteger)maxBreadcrumbs
         self.contextDictionary = [NSMutableDictionary new];
         self.attachmentArray = [NSMutableArray new];
         self.fingerprintArray = [NSMutableArray new];
+        self.attributesDictionary = [NSMutableDictionary new];
```

Comment:
`h`: The serialize method doesn't add the attributes. I just realized it doesn't make any sense that the scope implements the `SentrySerializable` protocol, so we could actually still ditch it in V9. If not, we should add the attributes to the serialize method, because otherwise it's quite confusing.